gammapy/makers/map.py

```diff
@@ -35,7 +38,7 @@ class MapDatasetMaker(Maker):
     """
 
     tag = "MapDatasetMaker"
-    available_selection = ["counts", "exposure", "background", "psf", "edisp"]
+    available_selection = ["counts", "exposure", "background", "psf", "edisp", "rad_max"]
```

Comment:
The `rad_max` option here is not really part of the data reduction as the other quantities. I think the `selection` rather refers to "which" data to reduce, while the `rad_max` rather refers to "how" it is reduced. In practice it is actually not optional at all, because once the IRFs are point like and the `rad_max` info is present one has to conduct the data reduction this way. So I would completely remove the option and not expose it to the user.